src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,20 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
-        return false;
+        int i;
+        {
+            do {
+
+                i = n % 10;
+                if (i==2)  return true;
+                n=n/10;
+
+            }
+            while (n < 1);
+
+
+        }
+
+       return false;
```

Comment:
All you need to solve this task is to think about :
1. how can you get rightmost digit in number ?
2. how can you remove rightmost digit from number ?

For example take number n=321:
1. get rightmost number of n -> 1
2. is 1 == 2 ? -> no
3. remove rightmost digit from n -> n=32
4. go to step 1.

And you should do this while n is greater than 0.